### DISCUSSION TITLE
Can no longer login

### DISCUSSION THREAD

**Discussion by evanmarshall-dev** [vercel/community#1971] [Help]
So I've been using my GitHub account for about a year now to deploy my web app to Vercel using vercel provided domain. 

I've recently added an analytics environment variable and a custom domain. The custom domain was associated with another test vercel account so I logged into that, removed the domain from that account and when I went back to log in into the web app vercel it now tells me: there is no vercel account linked with this GitHub account.

Nothing was deleted on the vercel account and I see that there is still a link when I log in to GitHub to verify.

I've tried deleting cookies, different devices and browsers and it still returns this error.

Please see attached.

Thank you.

![Screenshot_20230404_204306_Chrome Canary.jpg](https://user-images.githubusercontent.com/104649881/229946027-333dbfdd-7e51-4e2f-b187-b222bf7703ad.jpg)

**Comment by jsfowles** [comment]
I have this same error. I have about 10 sites I can't access right now.

**Comment by amyegan** [comment]
Hi @evanmarshall-dev. Is the account linked to the GitHub account you used for this post, or is there a different username that I should use to look into this problem?

**Reply by evanmarshall-dev** [reply]
Oh I believe this issue has been resolved. Thank you. I believe the vercel site was down at the time.
